php/ext/google/protobuf/encode_decode.c

```diff
@@ -1356,15 +1366,6 @@ void add_handlers_for_message(const void* closure, upb_handlers* h) {
     return;
   }
 
-  // Ensure layout exists. We may be invoked to create handlers for a given
```

Comment:
I don't understand why we can remove this

Reply:
This has been done in register_class